app/js/app/directives/graph_sketcher/GraphInput.js

```diff
@@ -1,7 +1,7 @@
-define(function(require) {
-    var templateUrl = require("/partials/graph_sketcher/graph_input.html");
-    
-    return ["$timeout", "$rootScope", "api", function($timeout, $rootScope, api) {
+"use strict";
+
+define(['/partials/graph_sketcher/graph_input.html'], function(templateUrl) {
+    return ["$timeout", "$rootScope", "api", function(_$timeout, $rootScope, _api) {
```

Comment:
@Morpheu5 - is there a reason to use the `_` prefix rather than just removing the name from the injection? I can see why for functions that take definite arguments you may want to do this, but for these injections surely it's best to just remove unnecesary names?

Reply:
Agreed, if we are not using them we can just get rid of them, they were probably used at some point or other but are no longer needed.